### DISCUSSION TITLE
Chia Win and earn relationship

### DISCUSSION THREAD

**Discussion by tzlsgr** [Chia-Network/chia-blockchain#4621] [Q&A]
Hello everyone,

i am searching for about 2 weeks how chia block win works and how it is connected with earnings.

I don't understand and i didnt find any answer for my question.

My question is that you earn chia only by winning or you earn by plotting and farming?

so if for example have 5 years to win means that i will earn 0.0020 chia/ day ? (based on calculations in chia calc site)
or if i do not with a block i will not earn at all?

And how is possible to tell if finally it is worth doing this?

i appreciate your help

thank you

**Comment by Muffexx** [comment]
Hey @tzlsgr,

you earn chia by farming. But plotting is the process needed to create the plots which are used for farming.
Additional basic information can be found here: [Wiki](https://github.com/Chia-Network/chia-blockchain/wiki/Beginners-Guide)

Unfortunately you wont earn 0.00020 chia/day. If one of your plots is eligible and if a proof is found in your plot you can win.
When you win you will be awarded 2 XCH (Chia).

The estimated time to win is calculated based on the current network space, that means that a growing network will further reduce your chance to win. Thus, the time to a win may differ greatly from the displayed value.

Whether you should continue to plot/farm is dependend on your opinion, your space and your patience.

kind regards
muffexx

**Reply by tzlsgr** [reply]
Hello @Muffexx ,

Thank you very much for your answer! It was really helpful to help me understand.
It took me quite a while to understand in depth for chia.
An other question that i can not find a clear answer is that how long does plotts are stored in permanent storage?
Some say that are stored for 5-10 days aprox.

Kind regards
tzlsgr

**Reply by lvcivs** [reply]
Plots are normal files and they stay on you disk until you delete them (or the disk fails).

**Reply by tzlsgr** [reply]
well....how long should i keep them? how long are they useful? if they dont win they are useless right?
btw i started chia mining and seems cool!

**Reply by SackofFluor** [reply]
the plots are (errors aside) good indefinitly.
So its not like a "throw away" lottery ticket. Its more like a ticket that you keep available all the time and when you number is drawn you can get lucky.

Thats why the "netspace" is increasing, people plot more and keep those plots available to the network.
